networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/validation/BlockValidator.java

```diff
@@ -87,7 +88,6 @@ public BlockValidator(RecentChainData recentChainData, StateTransition stateTran
                 }
               } catch (EpochProcessingException | SlotProcessingException e) {
                 LOG.error("BlockValidator: Unable to process block state.", e);
-                return InternalValidationResult.REJECT;
```

Comment:
I'd restore this.  While it is redundant today, it's not a good idea to catch and ignore an error then continue on. If code was later added after the catch block, the missing return here would likely go unnoticed and cause bugs.

Reply:
Makes sense. Done.